the-collab-lab/tcl-49-smart-shopping-list
# Do cp items error

## Description
Our goal with this task is to ensure that users are not able to add an empty item to their list or add the same item twice. If they try to do this, we need to show them an error message that explains the problem. This way, we’ll prevent some clutter in their lists.

## Related Issue

Closes #9 

## Acceptance criteria
- [x] Show an error message if the user tries to submit a new item that matches an existing name with punctuation and casing normalized. For instance, if the list contains `apples` and the user adds `aPples` or `apples,` or `apples.`.

	

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :insect: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates
[Demo](https://user-images.githubusercontent.com/68116083/200137182-f6ae3b91-ec2a-4cc3-bb06-2b40e76bd00f.webm)

## Testing Steps / QA Criteria
Please check the preview URL link here to see the feature on the Add Item page: https://tcl-49-smart-shopping-list--pr29-do-cp-items-error-2mg8i3em.web.app/
